packages/flutter_tools/lib/src/ios/cocoapods.dart

```diff
@@ -85,8 +86,15 @@ class CocoaPods {
     }
   }
 
-  /// Whether CocoaPods ran 'pod setup' once where the costly pods' specs are cloned.
-  Future<bool> get isCocoaPodsInitialized => fs.isDirectory(fs.path.join(homeDirPath, '.cocoapods', 'repos', 'master'));
+  /// Whether CocoaPods ran 'pod setup' once where the costly pods' specs are
+  /// cloned.
+  ///
+  /// A user can override the default location via the CP_REPOS_DIR environment
```

Comment:
Also comment on where was this specific env var name defined? Does cocoapods also have it hardcoded somewhere? I couldn't google it.

Reply:
https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/config.rb#L138

Unfortunately it's not very well documented, but there are a few issues on GitHub here and there about it.  Do you want a link to the source in here?